### ISSUE TITLE
Quit spewing millions of warning messages by default

### ISSUE THREAD

**Issue by ernfrid** [genome/bam-readcount#16]
The default value for -w should be 1 to prevent millions of warnings from spewing forth.

**Comment by colindaven** [comment]
Yep, true. I got > 20 30+ gig error logs running this on SLURM today. 

Set -w to 10 and it ran fine
